File: RSMK2/RSMK2/BranchOperationImpl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RSMK2
{
    public class BranchOperationImpl : MarshalByRefObject, BranchOperation
    {
        public static List<Branch> lst = new List<Branch>();

        public List<Branch> GetAll()
        {
            return lst;
        }

        public List<Branch> Add(Branch newItem)
        {
            lst.Add(newItem);
            return lst;
        }

        public List<Branch> DeleteAt(int index)
        {
            lst.RemoveAt(index);
            return lst;
        }
    }
}

File: zwsserver/src/operation/BranchOperationImpl.java
package operation;

import types.Branch;
import java.util.ArrayList;
import java.util.List;

public class BranchOperationImpl implements BranchOperation {

    public static List<Branch> lst = new ArrayList<Branch>();

    static {
        lst.add(new Branch("Береза", "описание"));
        lst.add(new Branch("Липа", "описание"));
    }

    @Override
    public List<Branch> GetAll() {
        return lst;
    }

    @Override
    public List<Branch> Add(Branch newItem) {
        lst.add(newItem);
        return lst;
    }

    @Override
    public List<Branch> DeleteAt(int index) {
        lst.remove(index);
        return lst;
    }
}
